packages/tax/src/services/tax-module-service.ts

```diff
@@ -112,7 +128,7 @@ export default class TaxModuleService<
       sharedContext
     )
 
-    const rates = regions.map((region, i) => {
+    const rates = regions.map((region: TaxRegionDTO, i: number) => {
```

Comment:
Doesn't taxRegionService_.create return a typed object? If so, the typings inside the map shouldn't be necessary as they will be inferred.